tests/integration/assessment/test_assessment.py

```diff
@@ -1,5 +1,118 @@
+# def test_table_inventory(ws, make_catalog, make_schema):
+#     pytest.skip("test is broken")
+
 import pytest
 
+from databricks.labs.ucx.mixins.compute import CommandExecutor
+
+
+@pytest.mark.skip(reason="Needs to have DLT Pipelines already created ")
+def test_pipeline_crawler(ws, wsfs_wheel, make_schema, sql_fetch_all):
+    _, inventory_database = make_schema(catalog="hive_metastore", schema="ucx_dk").split(".")
+    commands = CommandExecutor(ws)
+    commands.install_notebook_library(f"/Workspace{wsfs_wheel}")
+
+    commands.run(
```

Comment:
Why do you need to run this through command executor within dbr? Run it directly in client code with StatementExecutionBackend instead of RuntimeBackend